php_phongo.c

```diff
@@ -233,18 +233,32 @@ static void phongo_cursor_init(zval *return_value, mongoc_cursor_t *cursor, mong
 	intern = Z_CURSOR_OBJ_P(return_value);
 	intern->cursor = cursor;
 	intern->server_id = mongoc_cursor_get_hint(cursor);
-	intern->client = client;
+	intern->client = Z_MANAGER_OBJ_P(manager)->client;
+
+#if PHP_VERSION_ID >= 70000
+	ZVAL_COPY(&intern->manager, manager);
```

Comment:
Per the [`ZVAL_COPY()` definition](http://lxr.php.net/xref/PHP_7_0/Zend/zend_types.h#857), the ref count is incremented if the type is considered to be ref-counted.

This is effectively the same as `ZVAL_ZVAL(dst, src, 1, 0)` (yes copy, no dtor).